lib/morse_translator.rb

```diff
@@ -40,4 +42,25 @@ def initialize
                     " " => " "}
   end
 
+  def translate(message)
```

Comment:
I love this as a CEO method that is calling the #split_message and #use_dictionary methods!